src/routers/features/siciliaVola/auth.ts

```diff
@@ -0,0 +1,16 @@
+import { Router } from "express";
+import * as faker from "faker";
+import { addHandler } from "../../../payloads/response";
+import { addApiV1Prefix } from "../../../utils/strings";
+import { securedSvRouter } from "./secured";
+
+export const authSvRouter = Router();
+
+const addPrefix = (path: string) => addApiV1Prefix(`/mitvoucher/auth${path}`);
+
+/**
+ * Get the mit auth token
+ */
+addHandler(securedSvRouter, "get", addPrefix("/token"), (_, res) =>
+  res.json({ token: faker.random.uuid() })
```

Comment:
```suggestion
  res.json({ token: faker.datatype.uuid() })
```
this is deprecated ([I realized it just minutes ago](https://github.com/pagopa/io-dev-api-server/commit/579e19e6864e1df01ce69fcaa5872c888b18afd9#))

Reply:
@debiff fyi
I committed this change